plasmapy/formulary/collisions/helio/collisional_analysis.py

```diff
@@ -0,0 +1,351 @@
+"""
+Module containing the Collisional Analysis formulation.
+"""
+__all__ = ["thermalization_ratio"]
+
+import astropy.units as u
+import numbers
+import numpy as np
+
+from plasmapy.particles import Particle, ParticleLike
+from plasmapy.utils.decorators import validate_quantities
+
+
+@validate_quantities(
+    T_1={"can_be_negative": False, "equivalencies": u.temperature_energy()},
+    T_2={"can_be_negative": False, "equivalencies": u.temperature_energy()},
+)
+def thermalization_ratio(
+    r_0: u.au,
+    r_n: u.au,
+    n_1: u.cm**-3,
+    n_2: u.cm**-3,
+    v_1: u.m / u.s,
```

Comment:
```suggestion
def thermalization_ratio(
    *,
    r_0: u.au,
    r_n: u.au,
    n_1: u.cm**-3,
    n_2: u.cm**-3,
    v_1: u.m / u.s,
```
This would make everything keyword-only